extension/src/cli/dvc/version.ts

```diff
@@ -5,6 +5,27 @@ import {
 } from './constants'
 import { Toast } from '../../vscode/toast'
 
+export enum CliCompatible {
```

Comment:
[F] The return value from `isVersionCompatible` needs to provide more than a boolean value as we need more information on what message to give to the user. These are all of the possible situations that we can have when checking the CLI's version.